buildSrc/build.gradle.kts

```diff
@@ -24,14 +24,32 @@ repositories {
   mavenCentral()
 }
 
+tasks {
+  processResources {
+    duplicatesStrategy = DuplicatesStrategy.WARN
+  }
+}
+
+configurations {
```

Comment:
seems like maybe those root changes you applied to all are really relevant in the buildSrc vs the actual code?